squash/squash/settings/dev.py

```diff
@@ -4,6 +4,9 @@
 
 DEBUG = os.environ.get('DEBUG', 'True') == 'True'
 
+# Needed by django debug toolbar
+INTERNAL_IPS='127.0.0.1'
```

Comment:
that's used in development mode when the django debug toolbar is active, the debug toolbar only act on requests coming from the addresses listed in INTERNAL_IPS https://django-debug-toolbar.readthedocs.io/en/stable/configuration.html#toolbar-options